telegram/ext/filters.py

```diff
@@ -65,6 +68,15 @@ def __or__(self, other):
     def __invert__(self):
         return InvertedFilter(self)
 
+    def __str__(self):
+        # Do not rely on classes overwriting __init__ to set a name
+        # so we can keep backwards compatibility
```

Comment:
what about defining `name` as class attribute which defaults to `'<user_filter>'` (or something of the sort)?
then we can save the ugly code below...
Following that idea, you can then add `name` attribute to each of the inheriting classes which will reduce the need to call `super().__init__()` and modify all the lines instantiating the filters.